### ISSUE TITLE
QA Ph2_Accessibility - Mobile: Live Widget - Issue adjusting values using voice over instructions 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#398]
With voice over enabled on the Live Widget page, there is a discrepancy adjusting the width/height values.  Voice over says "swipe up or down with one finger to adjust the value", I've been unable to adjust the values with this method.  However, I have been able to adjust them with tapping twice and holding to drag the scale. 

Steps: 
1. Enable voice over
2. Go to https://phase2-dot-io-webapp-staging.appspot.com/io2016/
3. Expand menu > Select I/O Extended 
4. Scroll down and select Configure Widget 
5. Tap in the width or height scale 
6. Listen to voice over instructions 
7. Attempt to swipe up or down to adjust value 
8. Note issue 

Browser: 
Safari iPhone 6S iOS 9.2

![gio_398](https://cloud.githubusercontent.com/assets/7585604/13586635/f1a312b6-e476-11e5-88b9-d4354a6e2bc5.PNG)
